researchapp/fixtures.yml

```diff
@@ -30,14 +30,14 @@ class: !!python/name:researchapp.models.providers.OAuthClient
 args:
   client_id: research-app
   client_secret: demo-secret-s4s
-  scope: launch/patient patient/*.read offline_access
+  scope: patient/*.read offline_access
```

Comment:
http://syncfor.science/api-calls/ indicates our clients should all use "launch/patient"  -  why this deletion?